_docs/contribute/dev-process.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page-dev
+title: Overview of the FIO Development Process
+description: Overview of the Process for Contributing to FIO
+---
+
+# Overview of the FIO Community's Development Process
+The FIO Core team does NOT do scrum, sprint, or any strict develeopment process. That being said, a given development cycle works something like this:
+
+- Initiatives are discussed high up, and prioritized. 
+- From initiatives there is analysis and initial scoping (done by VP, and Ed and lead engineers)
+- From analysis an Epic and project plan is made.
+- Lead engineers handle each epic in jira. From the Epic, stories are assigned and executed (we dont do sub tasks on FIO, just stories).
+- Project status is tracked in a status board auto generated from contents of the each Epic.
+- Daily standups are performed.
+- It's up to the lead to decide whether to operate with a meeting based culture or asynchronous communications or whatever they wish. For example, some use asynchronous and try hard to do very few official meetings (just get on the phone and lets work)…but you get to do any process on your projects if you can get buy in on from Eric and your peers on the team.
+
+## FIO Team Vibe
```

Comment:
Remove this section. Again, the DAO model has multiple leads each with their own culture.

Reply:
removed